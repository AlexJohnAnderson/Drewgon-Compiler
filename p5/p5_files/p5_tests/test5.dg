int a;
bool b;
int c;
int foo(){
a =76;
b = true;
c = a * b;
return c;
}
