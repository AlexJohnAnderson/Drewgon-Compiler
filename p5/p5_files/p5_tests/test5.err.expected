FATAL [7,9]-[7,10]: Arithmetic operator applied to invalid operand
Type Analysis Failed
